ravenframework/Optimizers/GradientDescent.py

```diff
@@ -689,15 +717,16 @@ def _addToSolutionExport(self, traj, rlz, acceptable):
     """
     toAdd = {'stepSize': self._stepHistory[traj][-1]['magnitude']}
     for key, val in self._convergenceInfo[traj].items():
-      toAdd['conv_{}'.format(key)] = val
+      toAdd[f'conv_{key}'] = val
     # collect any additions from gradient and stepper
-    ## gradient
```

Comment:
I can see you have strong preferences on double comments; I use them as a heirarchy so I can see which comments are sub-bullets of other comments. I recommend trying to keep a light touch on other dev's style choices when going over code.

Reply:
Blame that on my PhD advisor, he was very particular about everything. I can put them back in if you really want them.

Reply:
Not worth reverting, just a note that may save you time next time!